tests/integration_tests/test_ingest_component.py

```diff
@@ -908,14 +908,6 @@ def test_ingest_same_programme_different_rounds(
     )
 
     assert len(Project.query.filter(Project.project_id == "HS-WRC-01").all()) == 2
-    assert (
-        Project.query.filter(Project.project_id == "HS-WRC-01").all()[0].programme_junction.submission.reporting_round
-        == 3
-    )
-    assert (
-        Project.query.filter(Project.project_id == "HS-WRC-01").all()[1].programme_junction.submission.reporting_round
-        == 4
-    )
```

Comment:
To pre-empt questions of why I removed this: this particular piece of code is not relevant to the actual purpose of the test, and is poorly written as there's no guarantee the 3rd round's project will always be above the 4th round.